Capri.Web/Configuration/DatabaseConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.EntityFrameworkCore;
+using Capri.Database;
+using Capri.Database.Entities.Identity;
+
+namespace Capri.Web.Configuration
+{
+    public static class DatabaseConfiguration
+    {
+        public static void AddDatabaseConfiguration(this IServiceCollection services, IConfiguration Configuration)
```

Comment:
Don't subordinate DatabaseConfiguration method to whole IConfiguration. Pass only needed information. In that case you can just pass one string.